src/features/landing/types/why-we/types.ts

```diff
@@ -0,0 +1,6 @@
+export default interface IWhyWeCardItem {
+  id: string | number;
```

Comment:
I think that we should avoid `any` or `|` types

he id property is currently defined as either a string or number. It's generally a good practice to keep id types consistent across your application. If there's no specific requirement for id to be both types, consider using just one (preferably string if these are meant to be unique identifiers, especially in a web context).